### DISCUSSION TITLE
Combo button for HOME menu

### DISCUSSION THREAD

**Discussion by LouisGarry** [aldostools/webMAN-MOD#1099] [Ideas]
Hello Aldo, how are you?  I have a little idea, as we know, the PS4 controller can be used on the PS3, but the HOME button doesn't work, so I'm thinking, can we add a combo button in wMM, for example L3 + R3 to display the HOME menu?  Thank you.

**Comment by aldostools** [comment]
You can set custom scripts to almost any combo available.

For `L3`+`R3` you should create the script `/dev_hdd0/tmp/wm_combo/wm_custom_l3_r3` with this command:
```
/pad.ps3?psbtn
```

NOTE: `L3`+`R3` combo is used for VshFpsCounter. Instead you could use `wm_custom_select_triangle` to press PS with `SELECT`+`△`

**Reply by Twojstarypijany98** [reply]
I'v made wm_custom_l2_triangle file (not txt) and i added it to /dev_hdd0/tmp/wm_combo and it doesnt work, maybe im doing something wrong or doesnt understand it all. Sorry im green to it. Pls help i dont want to use my phone to enter xmb menu to leave the game.

**Reply by aldostools** [reply]
That is not a valid file name for combos. There is a post at the bottom of this thread with the valid file names or check the Wiki
https://github.com/aldostools/webMAN-MOD/wiki/PAD-shortcuts-or-Combos#custom

**Reply by Twojstarypijany98** [reply]
![IMG_20250603_225402](https://github.com/user-attachments/assets/3f51a84b-adf2-47bd-bd2c-e48da9d6951f)


Shouldnt it look like this ??

**Reply by aldostools** [reply]
No. The path is correct, but the file name is not valid.

**Reply by Twojstarypijany98** [reply]
Okay so i renamed it r2_circle so it should work now ? Sorry this is my first time doing this iv only made custom psp

**Reply by Twojstarypijany98** [reply]
Someone should make yt video about this haha

**Reply by aldostools** [reply]
it should work, but you need to have SHOW IDPS checked in Combos, since these buttons are used for that command.

**Reply by aldostools** [reply]
> Someone should make yt video about this haha

true 🤣

**Reply by Twojstarypijany98** [reply]
Bro wtf  what i need to have ? What is show idps, i thought it easy but u make it harder with every minute

**Reply by Twojstarypijany98** [reply]
Brother if u wont help me, there is only left ps3 pad to buy :C

**Reply by aldostools** [reply]
Ok. Closed the discussion.

**Reply by Twojstarypijany98** [reply]
![Screenshot_2025-06-03-23-09-43-47_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/user-attachments/assets/6dd26b67-ce5e-4b9e-be86-b5b22afaacdb)


Like here, haha ?

**Comment by LouisGarry** [comment]
Thank you for replying, is there a combo for the HOME (PS) menu in wMM?

**Reply by aldostools** [reply]
No. Only the following combinations can be scripted. PS is not one.

```
/dev_hdd0/tmp/wm_combo/wm_custom_select_l3
/dev_hdd0/tmp/wm_combo/wm_custom_l3_r3
/dev_hdd0/tmp/wm_combo/wm_custom_l3_r3_l2
/dev_hdd0/tmp/wm_combo/wm_custom_l2_start
/dev_hdd0/tmp/wm_combo/wm_custom_r2_start
/dev_hdd0/tmp/wm_combo/wm_custom_select_l1
/dev_hdd0/tmp/wm_combo/wm_custom_select_r1
/dev_hdd0/tmp/wm_combo/wm_custom_select_circle
/dev_hdd0/tmp/wm_combo/wm_custom_select_triangle
/dev_hdd0/tmp/wm_combo/wm_custom_select_cross
/dev_hdd0/tmp/wm_combo/wm_custom_select_square
/dev_hdd0/tmp/wm_combo/wm_custom_select_left  
/dev_hdd0/tmp/wm_combo/wm_custom_select_right 
/dev_hdd0/tmp/wm_combo/wm_custom_select_up    
/dev_hdd0/tmp/wm_combo/wm_custom_select_down  
/dev_hdd0/tmp/wm_combo/wm_custom_l2_r2_select_start
/dev_hdd0/tmp/wm_combo/wm_custom_l1_r1_triangle
/dev_hdd0/tmp/wm_combo/wm_custom_l2_r2_triangle
/dev_hdd0/tmp/wm_combo/wm_custom_r2_triangle
/dev_hdd0/tmp/wm_combo/wm_custom_r2_circle
/dev_hdd0/tmp/wm_combo/wm_custom_r2_square
/dev_hdd0/tmp/wm_combo/wm_custom_select_r2_circle
```
